addons/spreadsheet_account/models/account.py

```diff
@@ -130,3 +231,51 @@ def get_account_group(self, account_types):
         )
         mapped = dict(data)
         return [mapped.get(account_type, []) for account_type in account_types]
+
+    @api.model
+    def get_residual_amount(self, args_list):
```

Comment:
Why is this not included in `spreadsheet_fetch_debit_credit`, the params are almost the same

Wouldn't it be at least possible to factorize some code?